lib/hex_web/user.ex

```diff
@@ -191,10 +192,20 @@ defimpl HexWeb.Render, for: HexWeb.User do
   import HexWeb.Util
 
   def render(user) do
-    user
-    |> Map.take([:username, :email, :inserted_at, :updated_at])
-    |> Map.update!(:inserted_at, &to_iso8601/1)
-    |> Map.update!(:updated_at, &to_iso8601/1)
-    |> Map.put(:url, api_url(["users", user.username]))
+    entity = user
+      |> HexWeb.Repo.preload(:owned_packages)
+      |> Map.take([:username, :email, :inserted_at, :updated_at, :owned_packages])
+      |> Map.update!(:inserted_at, &to_iso8601/1)
+      |> Map.update!(:updated_at, &to_iso8601/1)
+      |> Map.put(:url, api_url(["users", user.username]))
+
+
+    packages = Enum.map(entity.owned_packages, fn(package) ->
+        package
+        |> Map.take([:name])
+        |> Map.put(:url, api_url(["packages", package.name]))
+      end)
```

Comment:
Not sure if this is best practice but conventionally we do a json object where the name is the key instead of a list of json objects.